### PR TITLE
feat: replace hardcoded error message as constant

### PR DESCRIPTION
**Pull request by mkubdev**
## Description

Replaced error message duplication by constant.

## TODO

- [ ] add `ERR_INVALID_FLAGS_CONVERT` for `flags_str` variable initialization
- [ ] add `ERR_INVALID_KEY_CONVERT` for `encryption_config` variable initialization

### CODE REVIEW

**Review by darkterminal — COMMENTED** [review]
Good start my bro! :tada:

File: src/lib.rs [hunk 1]
```diff
@@ -4,6 +4,15 @@ use once_cell::sync::OnceCell;
...
 
+const ERR_NULL_CLIENT_PTR: &str = "Error: Client pointer is null";
+const ERR_PATH_IS_EMPTY: &str = "Error: Path is not defined";
+// const ERR_INVALID_FLAGS_CONVERT: &str = "Error: Failed to convert flags to string";
+const ERR_INVALID_PATH_CONVERT: &str = "Error: Failed to convert path to string";
+// const ERR_INVALID_KEY_CONVERT: &str = "Error: Failed to convert encryption key to string";
+const ERR_INVALID_ARGUMENTS: &str = "Error: Client pointer or query is null";
+const ERR_INVALID_QUERY_CONVERT: &str = "Error: Failed to convert query to string";
+const ERR_QUERY_EXECUTION: &str = "Error: Query execution failed";
+
```
Comment (darkterminal) [inline src/lib.rs:15]: Need to manage this in the future!

**Review by darkterminal — APPROVED** [review]
:tada:

File: src/lib.rs (hunk 1 above)
Comment (mkubdev) [reply src/lib.rs:15]: Indeed!

**Pull request merged**